### PR TITLE
Fix package metadata

### PR DESCRIPTION
**Pull request by Eilon**
@rynowak

### CODE REVIEW

**Comment by Eilon** [comment]
FYI this is for RC2/release.

File: src/Microsoft.AspNetCore.JsonPatch/project.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "version": "1.0.0-*",
+  "description": "ASP.NET Core support for JSON PATCH.",
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.JsonPatch/project.json]: One interesting point here, there isn't really anything ASP.NET about this library... it only depends on JSON.NET

**Comment by rynowak** [comment]
:shipit:

File: src/Microsoft.AspNetCore.JsonPatch/project.json (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.AspNetCore.JsonPatch/project.json]: Yeah but it's in the name 😄

**Comment by Eilon** [comment]
@ryanbrandenburg can you help me out and merge this to release, then merge back into dev as well?

**Comment by ryanbrandenburg** [comment]
Done.

**Comment by Eilon** [comment]
Thanks!!!

**Pull request merged**